interface_specifications/pe_openapi/apis/v1/d0_pe_presentation_post.md

```diff
@@ -0,0 +1,78 @@
+### Presentation POST
+
+`POST /pe/v1/{thid}/presentation/`
+
+
+#### Description
+
+Submit a requested presentation
+
+
+#### Parameters
+
+Path Variables:
+* `thid`: id of the presentation exchange thread e.g. `f1ca8245-ab2d-4d9c-8d7d-94bf310314ef`
+* `definition_id`: definition id for which this presentation is being submitted. e.g. `32f54163-7166-48f1-93d8-ff217bdb0653`
+
+
+Request Body:
+```json
+  {
+    "presentation_submission": {
```

Comment:
thid in the body (see earlier discussion)?

Reply:
fixed